api/controllers/AuthController.js

```diff
@@ -17,10 +17,19 @@ module.exports = {
 
     params = params.asObject();
 
+    var finduser;
+
     if (authConfig.wetland) {
       findUser = req.getRepository(sails.models.user.Entity).findOne({[loginProperty]: params[loginProperty]}, {populate: populate});
     } else {
-      findUser = sails.models.user['findOneBy' + _.upperFirst(loginProperty)](params[loginProperty]);
+      sails.models.user.findOne({username: 'test'}).exec(function (err, user){
```

Comment:
`findUser = sails.models.user.findOne({[loginProperty]: params[loginProperty]});`

Reply:
Undo the exec logic